cgmanifest.json

```diff
@@ -2141,6 +2141,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "httpd",
+          "version": "2.4.55",
```

Comment:
Instead of adding a new entry, please just update the existing one for the previous version (one entry above this one).